### PR TITLE
fix YC-880: archive requests with documents fails

### PR DESCRIPTION
**Pull request by olivierdalang**

### CODE REVIEW

File: geocity/apps/permits/tests/functional/test_complementary_documents.py [hunk 1]
```diff
@@ -207,3 +209,15 @@ def test_non_owners_can_not_delete_documents(self):
             messages.ERROR,
         )
         self.assertResponseMessageContains(actual, expected)
+
+    # regression test for YC-880
```
Comment (monodo) [inline geocity/apps/permits/tests/functional/test_complementary_documents.py:213]: Keep this comment here ?

Comment (olivierdalang) [reply geocity/apps/permits/tests/functional/test_complementary_documents.py:213]: I don't really have an opinion for that (there does not seem to be a standard practice for that in the codebase... could also put it as a docstring or remove it altogether)

Comment (monodo) [reply geocity/apps/permits/tests/functional/test_complementary_documents.py:213]: Looks to me that the test method name is explicit enough

**Comment by monodo** [comment]
fixed in phoenix :-)

**Pull request closed without merging**